source/server/server.cc

```diff
@@ -426,8 +428,8 @@ void InstanceBase::initialize(Network::Address::InstanceConstSharedPtr local_add
       });
 }
 
-void InstanceBase::initializeOrThrow(Network::Address::InstanceConstSharedPtr local_address,
-                                     ComponentFactory& component_factory) {
+absl::Status InstanceBase::initializeOrThrow(Network::Address::InstanceConstSharedPtr local_address,
```

Comment:
it seems like we can change the name from 'initializeOrThrow' to 'initialize'

Reply:
ah, I guess not, that probably the last step until we cleanup all the exception in all the deep layer~